src/main/webapp/components/index-cards/index-cards.js

```diff
@@ -107,14 +108,15 @@ export const IndexCardItem = props => {
           title={title}
           subheader={subHeader || moment(modified).fromNow()}
           action={headerAction}
+          titleTypographyProps={titleTypographyProps || ''}
```

Comment:
:pencil2: I don't think we should have the `|| ''` because that prop is expected to be an object or undefined